producer/src/contract/contract.go

```diff
@@ -26,8 +26,9 @@ func MakeYield(recipient *ecdsa.PublicKey, value uint64) Yield {
 }
 
 /* Inserts Yield into database */
-func InsertYield(y Yield, database string, blockHeight uint32, contractHash []byte, yieldIndex uint8) error {
+func InsertYield(y Yield, database string, blockHeight uint64, contractHash []byte, yieldIndex uint16) error {
```

Comment:
`yieldIndex` should not be a `uint16` there's no possible way we can support contracts with 65k yields.

Reply:
There's going to be more than 65k yields in a contract?

Reply:
No we should use a uint8. 65k is way too much.